.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "gems/base"]
+	path = gems/base
+	url = git@github.com:ActiveState/stackato-services-base.git
+	branch = 300049-password-leak-in-log-files
```

Comment:
@troytop, @philwhln  removed all the branches from the .gitmodules, to see the updates take a look at the individual pull request on each service.